Makefile

```diff
@@ -30,28 +30,32 @@ sync:
 	    tap-nikabot -c config.json --catalog catalog.json
 
 lint:
-	black -l 120 tap_nikabot tests *.py; \
-	flake8 --exit-zero tap_nikabot tests *.py; \
-	mypy --strict tap_nikabot; \
-	pylint tap_nikabot --disable 'broad-except,chained-comparison,empty-docstring,fixme,invalid-name,line-too-long,missing-class-docstring,missing-function-docstring,missing-module-docstring,no-else-raise,no-else-return,too-few-public-methods,too-many-arguments,too-many-branches,too-many-lines,too-many-locals,ungrouped-imports,wrong-spelling-in-comment,wrong-spelling-in-docstring,bad-whitespace'
+	source $(VENV_ACTIVATE); \
+		black -l 120 tap_nikabot tests *.py; \
+		isort -rc tap_nikabot tests *.py; \
+		flake8 --exit-zero tap_nikabot tests *.py; \
+		mypy --strict tap_nikabot || true
 
 test: lint
-	coverage run -m pytest && \
+	source $(VENV_ACTIVATE); \
+        coverage run -m pytest; \
```

Comment:
This and all other places above should use &&, not ;, otherwise the tests could fail but the coverage report being the last line is the only one whose exit code gets posted. This makes it so "make test" can fail the pytest command but still look "green" in circle.

Reply:
Good point, something to address in a future PR.